samples/dnn/person_reid.cpp

```diff
@@ -306,14 +306,20 @@ int main(int argc, char **argv)
             }
             if (imgWidth == -1){
                 imgWidth = min(image.rows, image.cols);
-                fontSize = (stdSize*imgWidth)/stdImgSize;
-                fontWeight = (stdWeight*imgWidth)/stdImgSize;
+                fontSize = min(fontSize, (stdSize*imgWidth)/stdImgSize);
+                fontWeight = min(fontWeight, (stdWeight*imgWidth)/stdImgSize);
             }
 
-            putText(image, "Press 's' to Draw Bounding Box, press 'Enter' after selecting", Point(10, 30), Scalar(0,0,0), fontFace, fontSize, fontWeight);
+            const string label = "Press space bar to pause video to draw bounding box.";
+            Rect r = getTextSize(Size(), label, Point(), fontFace, fontSize, fontWeight);
+            r.height += 2 * fontSize; // padding
+            r.width += 10; // padding
+            rectangle(image, r, Scalar::all(255), FILLED);
+            putText(image, label, Point(10, fontSize), Scalar(0,0,0), fontFace, fontSize, fontWeight);
+            putText(image, "Press space bar after selecting.", Point(10, 2*fontSize), Scalar(0,0,0), fontFace, fontSize, fontWeight);
             imshow("TRACKING", image);
             int key = waitKey(200);
-            if(key == 's' || key == 'S'){
+            if(key == ' '){
```

Comment:
I still don't see reaction on SPACE to start selection. Also, the message is misleading. Please, fix it.